django/db/migrations/state.py

```diff
@@ -545,6 +545,9 @@ def clone(self):
             app_label=self.app_label,
             name=self.name,
             fields=list(self.fields),
+            # Since we only shallow-copy options here, we need to ensure
```

Comment:
Avoid "we" in comments:
```
# Since options are shallow-copied here, operations such as
# AddIndex must replace their option (e.g 'indexes') rather than
# mutating it.
```